resources/energyplus/ProposedEnergy+.idd

```diff
@@ -42357,6 +42357,127 @@ Boiler:Steam,
+       \type object-list
+       \object-list ZoneNames
+  A7, \field Ambient Temperature Outdoor Air Node Name
+       \type node
+       \note required for Ambient Temperature Indicator=Outdoors
+  A8,  \field Chilled Water Inlet Node Name
+       \type node
+       \required-field
+  A9,  \field Chilled Water Outlet Node Name
+       \type node
+       \required-field
+  A10, \field Chilled Water Maximum Requested Flow Rate
+       \type real
+       \units m3/s
+       \minimum> 0
+       \autosizable
+       \ip-units gal/min
+       \default autosize
+  A11, \field Condenser Inlet Node Name
+       \type node
+  A12, \field Condenser Outlet Node Name
+       \type node
+  A13, \field Condenser Maximum Requested Flow Rate
+       \type real
+       \units m3/s
+       \minimum> 0
+       \autosizable
+       \ip-units gal/min
+       \default autosize
+  A14, \field Chiller Flow Mode
+       \note Select operating mode for fluid flow through the chiller. "NotModulated" is for
+       \note either variable or constant pumping with flow controlled by the external plant system.
+       \note "ConstantFlow" is for constant pumping with flow controlled by chiller to operate at
+       \note full design flow rate. "LeavingSetpointModulated" is for variable pumping with flow
+       \note controlled by chiller to vary flow to target a leaving temperature setpoint.
+       \type choice
+       \key ConstantFlow
+       \key LeavingSetpointModulated
+       \key NotModulated
+       \default NotModulated
+  A15, \field Oil Cooler Inlet Node Name
+       \note Use if the oil cooler uses an external cooling loop, otherwise the oil cooler will add
+       \note heat to the ambient conditions (i.e., it is air cooled).
+       \type node
+  A16, \field Oil Cooler Outlet Node Name
+       \type node
+  A17, \field Oil Cooler Design Flow Rate
+       \type real
+       \units m3/s
+       \minimum> 0
+       \ip-units gal/min
+  A18, \field Auxiliary Inlet Node Name
+       \note Use if the auxiliary components of the chiller use an external cooling loop, otherwise
+       \note the auxiliary components will add heat to the ambient conditions (i.e., they are air cooled).
+       \type node
+  A19, \field Auxiliary Outlet Node Name
+       \type node
+  A20, \field Auxiliary Cooling Design Flow Rate
+       \type real
+       \units m3/s
+       \minimum> 0
+       \ip-units gal/min
+  A21, \field Heat Recovery Inlet Node Name
+       \note Not yet implemented / reserved for future use. Heat recovery is not yet within scope of ASHRAE Stanard 205.
+       \type node
+  A21, \field Heat Recovery Outlet Node Name
+       \note Not yet implemented / reserved for future use. Heat recovery is not yet within scope of ASHRAE Stanard 205.
+       \type node
+  A23; \field End-Use Subcategory
+       \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
+       \type alpha
+       \retaincase
+       \default General
```

Comment:
New IDD object. Everything is required-field as much as possible.

**NOTE**:
* This object will eventually have **FIVE** loops: chilled water, condenser, heat recovery, oil cooler, and auxiliary
* Heat Recovery is reserved as a Tertiary connection, but it's disabled right now in model SDK due to the fact that E+ doesn't implement it yet.